apps/cyberstorm-storybook/package.json

```diff
@@ -40,6 +40,7 @@
     "@babel/core": "^7.16.0",
     "@babel/preset-env": "^7.16.11",
     "@babel/preset-typescript": "^7.16.0",
+    "@fortawesome/react-fontawesome": "^0.2.0",
```

Comment:
This seems like an unnecessary addition?

Reply:
I'm not sure at this point. Storybook doesn't work without this addition, for some reason. Although this doesn't belong in this PR. I'll make separate PR out of all the package version changes.

Reply:
Though I'll leave this here for now and try to get this same change merged into master before this.